conda_smithy/templates/azure-pipelines-linux.yml.tmpl

```diff
@@ -5,10 +5,30 @@
 jobs:
 - job: linux
   pool:
+  {%- if azure.pool is defined %}
+  # self-hosted agent
+    name: {{ azure.pool.name }}
+  {%- if azure.pool.demands is defined %}
+    demands:
+    {%- for requirement in azure.pool.demands %}
+    - {{ requirement }}
+    {%- endfor %}
+  {%- endif %}
+  {%- if azure.workspace is defined %}
+  workspace:
+    clean: {{ azure.workspace.clean }}
+  {%- endif %}
+  {%- else %}
+  # MS-hosted agent
     vmImage: ubuntu-16.04
+  {%- endif %}
   timeoutInMinutes: {{ azure.timeout_minutes }}
   strategy:
+    {%- if azure.strategy is defined and azure.strategy.maxParallel is defined %}
+    maxParallel: {{ azure.strategy.maxParallel }}
+    {%- else %}
     maxParallel: 8
+    {%- endif %}
```

Comment:
Should this be a parameter in `_load_forge_config` with a default parameter of `8`?

Reply:
Looks like it's a better approach! This would guarantee `azure.strategy.maxParallel` is always defined regardless of its definition (or lack thereof) in `conda-forge.yml`, right?

Reply:
yes